tests/integration/callbacks/test_callback_context.py

```diff
@@ -96,3 +99,247 @@ def report_triggered(n):
         'triggered is truthy, has prop/id ["btn", "n_clicks"], and full value '
         '[{"prop_id": "btn.n_clicks", "value": 1}]',
     )
+
+
+calls = 0
+callback_contexts = []
+clicks = dict()
+
+
+@pytest.mark.DASH1350
+def test_cbcx005_grouped_clicks(dash_duo):
+    app = Dash(__name__)
+    app.layout = html.Div(
+        [
+            html.Button("Button 0", id="btn0"),
+            html.Div(
+                [
+                    html.Button("Button 1", id="btn1"),
+                    html.Div(
+                        [html.Div(id="div3"), html.Button("Button 2", id="btn2")],
+                        id="div2",
+                        style=dict(backgroundColor="yellow", padding="50px"),
+                    ),
+                ],
+                id="div1",
+                style=dict(backgroundColor="blue", padding="50px"),
+            ),
+        ],
+        id="div0",
+        style=dict(backgroundColor="red", padding="50px"),
+    )
+
+    @app.callback(
+        Output("div3", "children"),
+        [
+            Input("div1", "n_clicks"),
+            Input("div2", "n_clicks"),
+            Input("btn0", "n_clicks"),
+            Input("btn1", "n_clicks"),
+            Input("btn2", "n_clicks"),
+        ],
+        prevent_initial_call=True,
+    )
+    def update(div1, div2, btn0, btn1, btn2):
+        global calls
+        global callback_contexts
+        global clicks
+
+        calls = calls + 1
+        callback_contexts.append(callback_context.triggered)
+        clicks["div1"] = div1
+        clicks["div2"] = div2
+        clicks["btn0"] = btn0
+        clicks["btn1"] = btn1
+        clicks["btn2"] = btn2
+
+    def click(target):
+        ActionChains(dash_duo.driver).move_to_element_with_offset(
+            target, 5, 5
+        ).click().perform()
```

Comment:
Just to be clear: the point here is to click in the padding of `target` so you're sure not to click any of its children?

Reply:
Exactly